global/prometheus-alertmanager-operated/values.yaml

```diff
@@ -84,6 +84,7 @@ slack:
 
 octobus:
   gcpInstance:
+  storage:
```

Comment:
We need to elaborate on this value. What is it actually?

The octobus values are globally set, what does storage relate to? Maybe we can find a better wording.